addon/server.js

```diff
@@ -38,17 +38,30 @@ function createPretender(server) {
 
     this.handledRequest = function(verb, path, request) {
       if (server.shouldLog()) {
-        console.log(`Mirage: [${request.status}] ${verb.toUpperCase()} ${request.url}`);
-        let { responseText } = request;
-        let loggedResponse;
+        console.groupCollapsed(
```

Comment:
Can you confirm that this works on mobile Safari and mobile Chrome? Compatibility [isn't clear](https://developer.mozilla.org/en-US/docs/Web/API/Console/groupCollapsed), and given that one may need to run tests against these targets it'd be nice to not have surprise fails here.